### ISSUE TITLE
training takes so long !

### ISSUE THREAD

**Issue by kotaxyz** [TEXTurePaper/TEXTurePaper#38]
hello there, 

i have been trying to do training but it take so long i have rtx 3060ti , my question is ,is it normal that training take so long and how much it takes to fininsh training on single gpu 

the log 

(texture) C:\Users\Genesis\github\texture>python -m scripts.finetune_diffusion --pretrained_model_name_or_path=stabilityai/stable-diffusion-2-depth --instance_data_dir=images/teapot_processed --instance_prompt="a photo of a teapot" --lr_warmup_steps=0 --max_train_steps=10000 --scale_lr --output_dir tuned_models/teapot_model --eval_path=configs/texture_transfer/eval_data.json --train_batch_size 1 --gradient_checkpointing --use_8bit_adam --gradient_accumulation_steps=2 --resolution 512 --num_train_epochs 1
C:\Users\Genesis\anaconda3\envs\texture\lib\site-packages\accelerate\accelerator.py:375: UserWarning: `log_with=tensorboard` was passed but no supported trackers are currently installed.
  warnings.warn(f"`log_with={log_with}` was passed but no supported trackers are currently installed.")
Using object as an initial word
You are using a model of type clip_text_model to instantiate a model of type . This is not supported for all configurations of models and can yield errors.
bin C:\Users\Genesis\anaconda3\envs\texture\lib\site-packages\bitsandbytes\libbitsandbytes_cuda113.dll
C:\Users\Genesis\anaconda3\envs\texture\lib\site-packages\diffusers\configuration_utils.py:215: FutureWarning: It is deprecated to pass a pretrained model name or path to `from_config`.If you were trying to load a scheduler, please use <class 'diffusers.schedulers.scheduling_ddpm.DDPMScheduler'>.from_pretrained(...) instead. Otherwise, please make sure to pass a configuration dictionary instead. This functionality will be removed in v1.0.0.
  deprecate("config-passed-as-path", "1.0.0", deprecation_message, standard_warn=False)
[WindowsPath('images/teapot_processed/1.jpeg'), WindowsPath('images/teapot_processed/2.jpeg'), WindowsPath('images/teapot_processed/3.jpeg'), WindowsPath('images/teapot_processed/4.jpeg'), WindowsPath('images/teapot_processed/5.jpeg')]
Steps:   0%|                                                                                                                                        | 0/10000 [00:00<?, ?it/s]a photo of a teapot [49406, 320, 1125, 539, 320, 40749, 49407]
a photo of a teapot [49406, 320, 1125, 539, 320, 40749, 49407]
Steps:   0%|                                                                                                                  | 0/10000 [00:21<?, ?it/s, loss=0.0161, lr=1e-5]a photo of a teapot [49406, 320, 1125, 539, 320, 40749, 49407]
Steps:   0%|                                                                                                       | 1/10000 [00:50<139:20:00, 50.17s/it, loss=0.127, lr=1e-5]a photo of a teapot [49406, 320, 1125, 539, 320, 40749, 49407]
Steps:   0%|                                                                                                     | 1/10000 [01:22<139:20:00, 50.17s/it, loss=0.00874, lr=1e-5]a photo of a teapot [49406, 320, 1125, 539, 320, 40749, 49407]
Steps:   0%|                                                                                                       | 3/10000 [02:27<130:06:30, 46.85s/it, loss=0.311, lr=1e-5]a photo of a teapot [49406, 320, 1125, 539, 320, 40749, 49407]
a photo of a teapot [49406, 320, 1125, 539, 320, 40749, 49407]
Steps:   0%|                                                                                                      | 3/10000 [02:59<130:06:30, 46.85s/it, loss=0.0187, lr=1e-5]a photo of a teapot [49406, 320, 1125, 539, 320, 40749, 49407]
Steps:   0%|                                                                                                       | 4/10000 [03:31<149:07:41, 53.71s/it, loss=0.041, lr=1e-5]a photo of a teapot [49406, 320, 1125, 539, 320, 40749, 49407]
Steps:   0%|                                                                                                      | 4/10000 [04:03<149:07:41, 53.71s/it, loss=0.0464, lr=1e-5]a photo of a teapot [49406, 320, 1125, 539, 320, 40749, 49407]

**Comment by eladrich** [comment]
We haven't tried training on a 3060ti so I can't tell. Generally, training a fine-tuned model can take a while, you might also not need to train for the specified number of steps. Check out the intermediate visualization to decide whether you can stop early
